Modules/_xxsubinterpretersmodule.c

```diff
@@ -627,6 +627,36 @@ _channelends_associate(_channelends *ends, int64_t interp, int send)
     return 0;
 }
 
+static int64_t *
+_channelends_list_interpreters(_channelends *ends, int64_t *count, int send)
+{
+    int64_t *ids = NULL;
+    int64_t numopen;
+
+    assert(ends != NULL);
+    assert(count != NULL);
+    numopen = send ? ends->numsendopen : ends->numrecvopen;
+
+    if (numopen >= INT64_MAX) {
+        PyErr_SetString(PyExc_RuntimeError,
+                        "too many interpreters using the channel");
+        goto done;
+    }
+    ids = PyMem_NEW(int64_t, (Py_ssize_t)numopen);
```

Comment:
```suggestion
    int64_t *ids = PyMem_NEW(int64_t, (Py_ssize_t)numopen);
```

...and if you get rid of `numopen`:

```suggestion
    int64_t *ids = PyMem_NEW(int64_t, (Py_ssize_t)*count));
```

Reply:
See reply to comment about using `*count` instead of `numopen`.